Source/WebCore/platform/mediastream/gstreamer/GStreamerWebRTCProvider.cpp

```diff
@@ -99,12 +101,13 @@ void GStreamerWebRTCProvider::initializeAudioDecodingCapabilities()
 void GStreamerWebRTCProvider::initializeVideoDecodingCapabilities()
 {
     m_videoDecodingCapabilities = GStreamerRegistryScanner::singleton().videoRtpCapabilities(GStreamerRegistryScanner::Configuration::Decoding);
-    m_videoDecodingCapabilities->codecs.removeAllMatching([isSupportingVP9Profile0 = isSupportingVP9Profile0(), isSupportingVP9Profile2 = isSupportingVP9Profile2()](const auto& codec) {
+    m_videoDecodingCapabilities->codecs.removeAllMatching([isSupportingVP9Profile0 = isSupportingVP9Profile0(), isSupportingVP9Profile2 = isSupportingVP9Profile2(), isSupportingH265 = isSupportingH265()](const auto& codec) {
         if (!isSupportingVP9Profile0 && codec.sdpFmtpLine == "profile-id=0"_s)
             return true;
         if (!isSupportingVP9Profile2 && codec.sdpFmtpLine == "profile-id=2"_s)
             return true;
-
+        if (!isSupportingH265 && codec.mimeType == "video/H265"_s)
+            return true;
         return false;
```

Comment:
Most stupid nit ever: for coherence with the audio version, you can put a blank like after the last `if` before the `return false`.